src/components/Cart/GroupCart/GroupCart.js

```diff
@@ -1,110 +1,213 @@
-                    <S.CartProduct>
-                      <S.CartProductSelect>
-                        <S.CartCheckboxWrap>
-                          <S.CartCheckbox type="checkbox" />
-                        </S.CartCheckboxWrap>
-                      </S.CartProductSelect>
-                      <S.CartSmallProduct>
-                        <S.CartProductImgWrap>
-                          <S.CartProductImg src={thumbnail} />
-                        </S.CartProductImgWrap>
-                        <S.CartProductContents>
-                          <S.CartProductTitle>{productName}</S.CartProductTitle>
-                          <S.CartProductDeliver>
-                            {delivery}
-                          </S.CartProductDeliver>
-                          <S.CartProductColor>{color}</S.CartProductColor>
-                        </S.CartProductContents>
-                      </S.CartSmallProduct>
-                      <S.CartDelete>X</S.CartDelete>
-                      <S.CartCountWrap>
-                        <S.CartCountArticle>
-                          <S.CartCountBox>
-                            <CartCount /> {/* count button 자리 */}
-                            <S.CartPriceWrap>
-                              <S.CartPrice>{price} 원</S.CartPrice>
-                            </S.CartPriceWrap>
-                          </S.CartCountBox>
-                        </S.CartCountArticle>
-                      </S.CartCountWrap>
-                    </S.CartProduct>
-                  </S.CartListGroup>
-                </S.CartArticleBox>
-                <S.CartFooter>
-                  <p>배송비 무료</p>
-                </S.CartFooter>
-              </S.CartArticle>
-            </S.CartList>
-          )
-        )}
-      </S.CartContainer>
       <S.CartSideBarWrap>
         <S.CartSideBarContainer>
           <S.CartSideBarBox>
             <S.CartSideBarContents>
               <S.CartTotalPrice>
                 총 상품금액
-                <S.CartTotalPriceNumber>1,565,000 원</S.CartTotalPriceNumber>
+                <S.CartTotalPriceNumber>
+                  {cartData
+                    .filter(item => checkList.includes(item.id))
+                    .reduce((acc, item) => acc + item.price * item.count, 0)
+                    .toLocaleString()}{' '}
+                  원
+                </S.CartTotalPriceNumber>
               </S.CartTotalPrice>
               <S.CartTotalDeliver>
-                총배송비
+                총 배송비
                 <S.CartTotalDeliverNumber>+ 0 원</S.CartTotalDeliverNumber>
               </S.CartTotalDeliver>
               <S.CartTotalDiscount>
                 총 할인금액
                 <S.CartTotalDiscountNumber>
-                  - 365,050 원
+                  -{' '}
+                  {cartData
+                    .filter(item => checkList.includes(item.id))
+                    .reduce((acc, item) => acc + item.discount * item.count, 0)
+                    .toLocaleString()}
```

Comment:
```suggestion
const discountedPrice =  cartData
                    .filter(item => checkList.includes(item.id))
                    .reduce((acc, item) => acc + item.discount * item.count, 0)
                    .toLocaleString()


return(
...
{discountedPrice}
```

브라우저에 할인된 가격이 랜더되야 하는거라면, 
복잡한 연산으로 나온 반환값을 변수에 저장해서,
변수를 참조하는 방식으로 분리 해 보세요.